packages/odyssey-react/src/components/Field/FieldError.tsx

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { forwardRef } from "react";
+import { withTheme } from "@okta/odyssey-react-theme";
+import type { ReactElement, ReactNode, FunctionComponent } from "react";
+import { CommonFieldProps } from "./types";
+import styles from "./FieldError.module.scss";
+
+export type FieldProps = CommonFieldProps & {
+  /**
+   * Input to be rendered within the Field
+   */
+  children: ReactElement | ReactElement[];
+
+  /**
+   * The underlying input element id attribute.
+   */
+  inputId: string;
+
+  /**
+   * The underlying parent semantic HTML element.
+   * @default div
+   */
+  as?: "div" | "fieldset";
+};
+
+interface FieldErrorProps {
+  id: string;
+  children: ReactNode;
+}
+
+export const FieldError: FunctionComponent<FieldErrorProps> = withTheme(
+  () => ({}),
+  styles
+)(
+  forwardRef<HTMLParagraphElement, FieldErrorProps>(({ children, id }, ref) => {
+    return (
+      <p className={styles.error} color="danger" id={`${id}-error`} ref={ref}>
```

Comment:
I think you mean to use `Box` here yes? I see you passing ` color` prop

Reply:
I had box here previously but removed it based on @edburyenegren-okta's recommendation here: https://github.com/okta/odyssey/pull/1235#discussion_r788320372

Reply:
Hm. Maybe that conversation was resolved; the link isn't helping me now :(

I'm not sure what I said before, but my expectation here would be:

`<Box as="p" className={styles.error} id={`${id}-error`} ref={ref}>`

I'm not sure if there are any Box styles we'd need to opt out of here. Color? 

That said, my expectation here represents preference.